frontend/src/components/Paper/PaperCard/PaperCard.js

```diff
@@ -46,9 +47,9 @@ class PaperCard extends Component {
     ).slice(0, 10).filter(
         (keyword) => keyword.type === type,
     ).map(
-        (keyword) => (
+        (keyword, index) => (
```

Comment:
Well, can this be solution? Because index of arrays can be duplicated in many PaperCards. Did you checked there is no warning about this?(not Eslint)

Reply:
If we can only focus on duplication of one PaperCard, I can’t understand your problem. Maybe more explanation is needed.

Reply:
Yes when i changed id to index, no error occurred because the index is related to map itself.

Reply:
Got it. However, I can't understand why the problem occurs in this PR.